app/views/layouts/application.html.erb

```diff
@@ -36,7 +36,7 @@
 <head>
   <title><%= yield :title %> - <%= t('application.title.suffix')%></title>
   <%= javascript_include_tag "test-dependencies" if Rails.env.test? %>
```

Comment:
I wonder if we should also add type="module" here as well. It appears that test-dependencies.js is used to pull in [JS used by static to replicate the approach used on dev/production](https://github.com/alphagov/frontend/blob/main/app/assets/javascripts/test-dependencies.js#L1-L2), not 100% sure if this would actually make any difference to the tests, but would just align the approach used (to be used) more closely.